### DISCUSSION TITLE
Ffcutter to create chunks of mp3

### DISCUSSION THREAD

**Discussion by pramatias** [GrandaddyShmax/audiocraft_plus#17] [General]
Hello, i created ffcutter, to automatically cut mp4 output into smaller chunks to give as a melody source into audiocraft. [1]

A question i would like to ask, if anyone knows, do wav files work better as a melody source than mp4 converted to mp3? 

I don't know anything about audio processing, but mp3 files work fine by my experience. I was wondering, do wav files converted in some special way to mp3 or ogg, may provide some advantage.  

https://github.com/pramatias/ffcutter

**Comment by GrandaddyShmax** [comment]
It is hard to say if wav is better than mp4 but from my testing it did sound that wav produced much cleaner result
